src/main/java/aquality/selenium/core/utilities/IElementActionRetrier.java

```diff
@@ -4,13 +4,15 @@
 import org.openqa.selenium.StaleElementReferenceException;
 
 import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
 import java.util.function.Supplier;
 
 /**
- * Retries an action or function when {@link #getHandledExceptions()} occurs.
+ * It was renamed to IActionRetrier.
```

Comment:
Why do we need this comment?

Reply:
I didn't finish of fixes for the moment. Already fixed.